files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -101,16 +101,30 @@ function check_list()
     echo ${RET_LIST}
 }
 
+function set_cpufreq_governor() {
+    governor="$1"
+    echo "$governor" | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor \
```

Comment:
@stepanblyschak tee `stdout` is redundant. Can we hide it?

Reply:
@nazariig  done